Healthy/Classes/ReusableViews/FormTextField/FormTextField.swift

```diff
@@ -0,0 +1,78 @@
+
+import UIKit
+
+class FormTextField: UIView {
+    
+    // MARK: - Outlets
+    //
```

Comment:
```suggestion
    // MARK: Outlets
```
We don't usually use MARK with separator inside the context